sdks/java/io/kinesis/src/test/java/org/apache/beam/sdk/io/kinesis/KinesisServiceMock.java

```diff
@@ -27,7 +27,7 @@
 
 /** Simple mock implementation of Kinesis service for testing, singletone. */
 public class KinesisServiceMock {
-  private static KinesisServiceMock instance;
+  private static volatile KinesisServiceMock instance;
```

Comment:
Wow I am amazed error-prone report this, respect to this static analysis.
However maybe it is simpler to define just a synchronized accessor as they recommend here:
http://errorprone.info/bugpattern/DoubleCheckedLocking